modules/javafx.graphics/src/main/java/javafx/animation/Timeline.java

```diff
@@ -46,21 +46,21 @@
  * to {@code Timeline}'s initial position, depends on {@code Timeline}'s
  * direction.
  * <p>
- * {@code Timeline} processes individual {@code KeyFrame} at or after specified
- * time interval elapsed, it does not guarantee the timing when {@code KeyFrame}
+ * {@code Timeline} processes individual a {@code KeyFrame} at or after the specified
```

Comment:
The `a` and `KeyFrame` are switched in this sentence; it should be `processes an individual @{KeyFrame}` instead.

Also, the previous paragraph should say either `processes individual {@code KeyFrame}s sequentially` or  `processes individual {@code KeyFrame} objects sequentially` or similar.